src/components/CodeInput/CodeInput.tsx

```diff
@@ -30,12 +30,16 @@ interface CodeInputProps {
 
   /** render custom view for cursor/indicator */
   customCursor?: () => ReactNode
-
   /** render custom view for cursor/indicator */
   secureTextEntry?: boolean
 
   /** keyboard type */
   keyboardType?: KeyboardTypeOptions
+
+  withCursor?: boolean
+
+  placeholder?: string
+  placeholderTextColor?: ColorValue
```

Comment:
why do you use type `ColorValue` ? 
i think you should use type `string | keyof theme color`